node/perf_hooks.ts

```diff
@@ -1,5 +1,40 @@
-export const { performance, PerformanceEntry } = globalThis as
-  & typeof globalThis
-  & {
+import { notImplemented } from './_utils'
```

Comment:
this import specifier needs to be "./_utils.ts"

Reply:
Done 👍

Reply:
Thanks for updating, but I meant you can't omit '.ts' extension (which causes an error in CI). In deno, file extensions need to be fully specified.

Reply:
Sorry, about that, i am kind of very new to deno 😅

Reply:
No problem. Thanks for updating!